dev/values/validators/all-without-native.yml

```diff
@@ -22,7 +22,7 @@ genesis:
 
 sequencer:
   upgrades:
-    upgrade1:
+    aspen:
```

Comment:
just realized we should be able to cleanup the upgrades section and price feed sections here, missed this when i was updating the other parts.